curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/6708382cf088b216580a9ff1.md

```diff
@@ -2,7 +2,7 @@
 id: 6708382cf088b216580a9ff1
 title: What Are IDs and Classes, and When Should You Use Them?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: yTuzVH6THdg
```

Comment:
this should be reverted back to the placeholder.
we only want to link to the freeCodeCamp concepts channel. this is Ania's personal youtube channel